.github/workflows/ci-latest-slang.yml

```
@@ -19,9 +19,18 @@ permissions:
   statuses: write
 
+concurrency:
+  group: ${{ github.event_name == 'repository_dispatch' && format('slang-pr-{0}', github.event.client_payload.slang_pr_number) || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   build:
     runs-on: ${{ matrix.runs-on }}
+    # For repository_dispatch (Slang PR testing), only run Linux and Windows Release.
+    # Debug and macOS are covered by nightly runs.
+    if: >-
```

actionlint (1.7.11)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)